### PR TITLE
[FIX] web_editor: fix dropdown positioning when toolbar at bottom of …

### PR DESCRIPTION
**Pull request by deso-odoo**
**Current behavior before PR:**

When the toolbar was located at the bottom of the screen, the dropdown menus would open outside of the viewport.

**Desired behavior after PR is merged:**

When the toolbar is in the bottom of the screen the dropdown will always open upwards ensuring they remain visible within the viewport.

**Task** - 3149341

### CODE REVIEW

File: addons/mass_mailing/static/src/scss/mass_mailing.ui.scss [hunk 1]
```diff
@@ -301,6 +301,28 @@ body.editor_enable.o_basic_theme.o_in_iframe {
...
+                }
+            }
+            // Popper.js manipulates transform and position of the dropdown. In
+            // mass_mailing, its calculations are wrong. These !important rules
+            // removes the wrong values automatically generated by Popper.
+            #justify , #style.dropdown, #font-size.dropdown {
+                > .dropdown-menu {
+                    transform: unset !important;
+                    bottom: 100% !important;
+                    top: unset !important;
```
Comment (Zinston) [inline addons/mass_mailing/static/src/scss/mass_mailing.ui.scss:321]: I'm confused, I thought @morm-odoo had done pretty much the exact same fix a couple of months ago. Was it in 16.0 ?

Comment (deso-odoo) [reply addons/mass_mailing/static/src/scss/mass_mailing.ui.scss:321]: @Zinston Yes this fix is done in 16.0 and not in 15.0.

Comment (Zinston) [reply addons/mass_mailing/static/src/scss/mass_mailing.ui.scss:321]: Thanks, that clarifies it. Sorry for budding in, carry on :-D

File: addons/web_editor/static/src/scss/wysiwyg_snippets.scss [hunk 2]
```diff
@@ -246,6 +246,14 @@ body.editor_enable.editor_has_snippets {
...
     &.o_loaded {
         transform: none;
+        > .o_we_customize_panel .oe-toolbar {
+            #style.dropdown, #font-size.dropdown {
+                > .dropdown-menu {
+                    overflow-y: scroll;
+                    max-height: 280px;
+                }
+            }
+        }
```
[the remark this answers is not in the document]
Comment (deso-odoo) [reply addons/web_editor/static/src/scss/wysiwyg_snippets.scss:256]: @dmo-odoo No this was not a part of the original commit, I added it as the current fix was breaking the dropdown in the website. I have attached a screenshot for reference.
after
before

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**